### PR TITLE
clean up the gapfilling code

### PR DESCRIPTION
**Pull request by exaexa**

### CODE REVIEW

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit ee5d62e8 is formatted properly.

File: test/reconstruction/gapfill_minimum_reactions.jl [hunk 1]
```diff
@@ -2,15 +2,9 @@
...
-    m2 = Metabolite("m2")
-    m3 = Metabolite("m3")
-    m4 = Metabolite("m4")
-    m5 = Metabolite("m5")
-    m6 = Metabolite("m6")
-    m7 = Metabolite("m7")
-    m8 = Metabolite("m8")
+    model = StandardModel("partial model")
+
+    (m1, m2, m3, m4, m5, m6, m7, m8) = Metabolite.("m$i" for i = 1:8)
```
Comment (stelmo) [inline test/reconstruction/gapfill_minimum_reactions.jl:7]: wow, didn't know this is possible

Comment (exaexa) [reply test/reconstruction/gapfill_minimum_reactions.jl:7]: I actually wanted to just use the array later in reactions but apparently `m[1] → m[2]` syntax doesn't work there.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit b23bb468 is formatted properly.

**Review by stelmo — APPROVED** [review]
Once tests pass, this looks gr8

File: test/reconstruction/gapfill_minimum_reactions.jl (hunk 1 above)
Comment (stelmo) [reply test/reconstruction/gapfill_minimum_reactions.jl:7]: that would be nice actually... later we need to rethink how the user manually enters reactions and metabolites, to make it better

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit c68317b4 is formatted properly.

**Comment by cylon-x** [comment]
:green_circle: &nbsp;Commit 605d8d1d is formatted properly.

**Pull request merged**